apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/SpringConfiguration.java

```diff
@@ -363,6 +371,37 @@ public MetricRegistry metricRegistry() {
     final JmxReporter reporter = JmxReporter.forRegistry(metricRegistry).build();
     reporter.start();
 
+    String newRelicMetricKey = System.getenv("NEW_RELIC_METRIC_KEY");
+
+    if (newRelicMetricKey != null) {
+      String newRelicAppName = System.getenv("NEW_RELIC_APP_NAME");
+
+      String hostname;
+      try {
+        hostname = InetAddress.getLocalHost().getHostName();
+      } catch (UnknownHostException e) {
+        hostname = "unknown";
+      }
+
+      SenderConfiguration configuration =
+          SenderConfiguration.builder("https://gov-metric-api.newrelic.com", "/metric/v1")
+              .httpPoster(new OkHttpPoster())
+              .apiKey(newRelicMetricKey)
+              .build();
+
+      MetricBatchSender metricBatchSender = MetricBatchSender.create(configuration);
+
+      Attributes commonAttributes =
+          new Attributes().put("host", hostname).put("appName", newRelicAppName);
+
+      NewRelicReporter newRelicReporter =
+          NewRelicReporter.build(metricRegistry, metricBatchSender)
+              .commonAttributes(commonAttributes)
+              .build();
+
+      newRelicReporter.start(15, TimeUnit.SECONDS);
```

Comment:
15 seconds seems very reasonable to me.

Reply:
Since 15 seconds is a bit arbitrary, again it might be good to set this via an environment variable.  Then if it is too frequent / too infrequent you can change it without having to change code.

Reply:
I guess that comment really depends on the conventions BFD typically uses.  I tend to favor configurablility over hardcoded.  One thing I often do is have a sane default (15) but allow that to be overridden via an environment variable.

This approach could be used here and above for the URL.  That way you don't have to set them, but you can if you need to at some point in the future.

Reply:
Good point on the timer. I'll add in the variable with the default staying at 15. What's the Java way to do that? In other languages, I'll often see something like `YOUR_VAR || 15`. Is it a full on if block to check if the env var is null?

Reply:
Just going to catch any parsing exceptions and set the default there instead. Save myself a step.